src/Tribe/Service_Provider.php

```diff
@@ -63,7 +63,6 @@ public function register() {
 	 */
 	protected function hook() {
 
-		//todo change these to only load when required during Sprint 4
```

Comment:
Is this addressed?

Reply:
I left only the ones that have hooks that should run:
https://github.com/mt-support/tribe-ext-hubspot/blob/feature/137548-add-dev-hapi-key/src/Tribe/Service_Provider.php#L64

Let me know if there is a better way to do this. Thanks